src/main/java/db/migration/V192__Assign_Instructor_Types.java

```diff
@@ -0,0 +1,225 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import java.sql.*;
+
+public class V192__Assign_Instructor_Types implements JdbcMigration {
+    static Long INSTRUCTOR_ROLE = 15L;
+
+    static Long EMERITI = 1L;
```

Comment:
Add a comment on what these are, e.g. "Instructor types"

Reply:
Are these instructor types hard-coded to be these numbers in the database or are you relying on a MySQL auto-increment key having a certain value?

Reply:
I am relying on their auto-increment key having the expected value (from the insert statements. It didn't seem wise to key off their description value, though that would also be unique.